packages/@aws-cdk/aws-cognito/test/integ.user-pool-signup-code.ts

```diff
@@ -0,0 +1,50 @@
+import { App, CfnOutput, Stack } from '@aws-cdk/core';
+import { UserPool, UserPoolClient, VerificationEmailStyle } from '../lib';
+
+/*
+ * Stack verification steps:
```

Comment:
Soon @shivlaks will introduce support for writing stack outputs to a file so this can be just a simple script...